Movecraft/src/main/java/net/countercraft/movecraft/async/AsyncManager.java

```diff
@@ -293,7 +293,7 @@ private void processSinking() {
             if (!(craft instanceof SinkingCraft))
                 continue;
 
-            if (craft.getHitBox().isEmpty() || craft.getHitBox().getMinY() < 5) {
+            if (craft.getHitBox().isEmpty() || craft.getHitBox().getMinY() < (craft.getWorld()getMinHeight() +5 ) {
```

Comment:
you missed a `.`

Reply:
Oopse my bad, should be fixed now.

Reply:
also fixed a missing `)`. sorry first time contributing PRs... be gentle.